src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -0,0 +1,392 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Transforms;
+using Microsoft.ML.Transforms.TensorFlow;
+
+[assembly: LoadableClass(TensorFlowTransform.Summary, typeof(IDataTransform), typeof(TensorFlowTransform.Arguments), typeof(SignatureDataTransform),
```

Comment:
>LoadableClass [](start = 11, length = 13)

I think this one is broken. Did you try accessing TF via commandline? 
I think it should be `typeof(IDataTransform), typeof(TensorFlowTransform), typeof(TensorFlowTransform.Arguments), typeof(SignatureDataTransform),`